extensions/ApiExplorer/SpecialApiExplorer.php

```diff
@@ -50,7 +50,7 @@ public function execute( $par = null ) {
 		wfProfileIn( __METHOD__ );
 
 		// TODO: Make this work for ResourceLoader (Wikia isn't using RL yet at the time of this writing).
-		// Wikia has the cachebuster in the wgExtensionPath (we rewrite that in varnish because many proxies won't cache things that have "?" in the URL), but other MediaWikis need the style-version in the querystring.
+		// Wikia interpolates wgStyleVersion in the wgExtensionPath (we rewrite that in varnish because many proxies won't cache things that have "?" in the URL), but other MediaWikis need the style-version in the querystring.
 		$cbSuffix = ( isset( $wgCityId ) ? "?{$wgStyleVersion}" : "" );
 		$wgOut->addScript( "<script type=\"text/javascript\" src=\"{$wgExtensionsPath}/wikia/JavascriptAPI/Mediawiki.js{$cbSuffix}\"></script>" );
 		$wgOut->addScript( "<script type=\"text/javascript\" src=\"{$wgExtensionsPath}/ApiExplorer/apiExplorer.js{$cbSuffix}\"></script>" );
```

Comment:
```
> echo "<script type=\"text/javascript\" src=\"{$wgExtensionsPath}/ApiExplorer/apiExplorer.js{$cbSuffix}\"></script>"
<script type="text/javascript" src="https://slot1-images.wikia.nocookie.net/__cb123/common/extensions/ApiExplorer/apiExplorer.js"></script>
```

Let's get rid of `$cbSuffix` variable.

Reply:
:white_check_mark: , see f9505c0fe1701510c7d75b0b6b934b9559a086c7